lib/restful_client_configuration.rb

```diff
@@ -45,11 +44,10 @@ def reset
 
   ## Dummy method to test reporting phase
   def report_on
-    @report_method.call("RestfulClientConfiguration", "Initialized at: #{Time.now}.")
+    @report_method.call('RestfulClientConfiguration', "Initialized at: #{Time.now}.")
```

Comment:
Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.now.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.to_i`, `Time.now.to_f` instead.